config/initializers/resque.rb

```diff
@@ -1 +1 @@
-Resque.inline = ENV['RAILS_ENV'] == "development"
+Resque.inline = ENV['RAILS_ENV'] == "development" || ENV['JOB_SYNC']
```

Comment:
We added this, based on our conversation yesterday, to run the job synchronous or not.